connectors/spark/src/main/scala/io/unitycatalog/spark/OptionUtil.java

```diff
@@ -0,0 +1,24 @@
+package io.unitycatalog.spark;
+
+import java.util.Map;
+
+public class OptionUtil {
+  private OptionUtil() {
+  }
+
+  public static final String URI = "uri";
+  public static final String TOKEN = "token";
+  public static final String WAREHOUSE = "warehouse";
+
+  public static final String RENEW_CREDENTIAL_ENABLED = "renewCredential.enabled";
+  public static final boolean DEFAULT_RENEW_CREDENTIAL_ENABLED = true;
```

Comment:
please take note that if we backport this to 0.3, the default should be false.

Reply:
That's true,  let me revert this back to false.

Reply:
Thanks for the comment !